man/QDNAseq-package.Rd

```diff
@@ -32,10 +32,13 @@
 
 \section{How to cite this package}{
     Whenever using this package, please cite:
-    Scheinin I, Sie D, Bengtsson H, van de Wiel MA, Olshen AB, van Thuijl HF, van Essen HF, Eijk PP, Rustenburg F, Meijer GA,
-Reijneveld JC, Wesseling P, Pinkel D, Albertson DG and Ylstra B (2014). "DNA copy number analysis of fresh and
-formalin-fixed specimens by shallow whole-genome sequencing with identification and exclusion of problematic regions in the
-genome assembly." _Genome Research_, *24*, pp. 2022-2032.
+    Scheinin I, Sie D, Bengtsson H, van de Wiel MA, Olshen AB, van Thuijl
+HF, van Essen HF, Eijk PP, Rustenburg F, Meijer GA, Reijneveld JC,
+Wesseling P, Pinkel D, Albertson DG and Ylstra B (2014). "DNA copy
+number analysis of fresh and formalin-fixed specimens by shallow
+whole-genome sequencing with identification and exclusion of
+problematic regions in the genome assembly." _Genome Research_, *24*,
+pp. 2022-2032.
```

Comment:
I suspect newline differences, LF vs CRLF. On the other hand I think I account for that in Rdoc but not sure. Also, I don't experience this moving between Windows and Linux myself. Could also be differences in our settings on how Git Deals with newlines.

Side note: I'm now ready to suggest roxygen2 for all new projects. It's stable and supports use cases I was missing in the past. (Still not handling setMethodS3() though)